source/GameAction.cpp

```diff
@@ -182,6 +182,22 @@ void GameAction::LoadSingle(const DataNode &child)
 		else
 			child.PrintTrace("Error: Skipping invalid \"fine\" with non-positive value:");
 	}
+	else if(key == "mortgage" && hasValue)
+	{
+		mortgages.emplace_back(max<int64_t>(0, child.Value(1)));
+		GameAction::MortgageInfo &mortgage = mortgages.back();
```

Comment:
It would appear so.
![image](https://github.com/endless-sky/endless-sky/assets/17688683/93837ef3-9ff3-4a0d-900c-a2a8710ef7e5)
We have a few spots we could update to use this now.